skbio/stats/distance/_regression.py

```diff
@@ -0,0 +1,170 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013--, scikit-bio development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+
+import six
+import numpy as np
+from skbio.util._decorator import experimental
+from skbio.util._misc import check_random_state
+from skbio.stats.distance._mantel import _order_dms
+
+from skbio.stats.distance import DistanceMatrix
+
+@experimental(as_of="0.4.0")
+def linregress(y, *args, **kwargs):
```

Comment:
Is this method implementing [MRM](http://mb3is.megx.net/gustame/constrained-analyses/multiple-regression-on-dissimilarity-matrices) or [MLR](https://sites.google.com/site/mb3gustame/constrained-analyses/multiple-regression)? It sounds like the latter but you mentioned checking the output against ecodist's MRM function. Just want to make sure I'm on the same page :)